### DISCUSSION TITLE
Downloaded video file title error

### DISCUSSION THREAD

**Discussion by KDB12345** [aclap-dev/video-downloadhelper#135] [Help and Q&A]
### Your question

Originally, when downloading a video from a video site, the video title was automatically downloaded as a file name. But from some point on, the download file name consists of only numbers. For example, in the past, the download video file name was 'Football  Highlight', but now it is downloaded like '328585_480p'. I think it's because of the recent update, but I hope it comes out as a video title like in the past.

**Comment by Astraeus000** [comment]
I'm having a similar issue with the download naming, but in my case it's a bit stranger than that even. Anything I download ends up named "getvid" and is classified as just a generic "file" in the DL folder. If I click on it Windows asks me what program to use to open the file, and it CAN be opened with a video player and it'll work fine. If I rename the file with the .mp4 extension as part of the filename then it turns into a normal mp4 and is then categorized as such.

Downloads still work fine and all, but if I were trying to get multiple files at once this would start to become a rather annoying issue.

Edit; Included some screenshot clips of exactly what happens.
![Untitled](https://github.com/aclap-dev/video-downloadhelper/assets/160296343/67e4489c-1573-4526-8614-4adedea289eb)

**Comment by paulrouget** [comment]
Hi! We are aware of that issue, and it will be fixed in the upcoming version (this weekend).

Sorry for the bug :|
